.circleci/config.yml

```diff
@@ -135,6 +135,11 @@ jobs:
           environment:
             SUPPRESS_GUTENBERG_MOBILE_JS_BUNDLE_BUILD: 1
           command: ./gradlew :libs:WordPressProcessors:test --stacktrace
+      - run:
+          name: Test ImageEditor
+          environment:
+            SUPPRESS_GUTENBERG_MOBILE_JS_BUNDLE_BUILD: 1
```

Comment:
`SUPPRESS_GUTENBERG_MOBILE_JS_BUNDLE_BUILD` is no longer necessary. It looks like this branch might have been created from an older commit since it still has Gutenberg related code in the CI config.

```suggestion
```